libs/ui/elements/src/lib/markdown-parser/markdown-parser.component.ts

```diff
@@ -0,0 +1,16 @@
+import { Component, Input, ViewEncapsulation } from '@angular/core'
+import { marked } from 'marked'
+
+@Component({
+  selector: 'gn-ui-markdown-parser',
+  templateUrl: './markdown-parser.component.html',
+  styleUrls: ['./markdown-parser.component.css'],
+  encapsulation: ViewEncapsulation.None,
```

Comment:
you can get rid of this and leave the default view encapsulation by adding `:host /deep/` before all classes in the CSS file; this is a bit more verbose but IMO essential since disabling view encapsulation is risky in the long run